src/pubtools/_marketplacesvm/tasks/community_push/command.py

```diff
@@ -305,10 +305,13 @@ def enrich_mapped_items(
                 for dest in mrobj.destinations:
                     pi = mapped_item.get_push_item_for_destination(dest)
                     log.debug("Mapped push item for %s: %s", storage_account, pi)
-                    beta = self.args.beta or str(pi.release.type) == "beta"
+                    r = dest.meta.get("release") or {}
+                    r_type = str(r.get("type", "")).lower()
+                    beta = self.args.beta or r_type == "beta"
                     epi = enrich_push_item(
                         pi,
                         dest,
+                        ga=r_type == "ga",
                         beta=beta,
```

Comment:
Instead of 2 parameters we can pass `release_type` directly :-)
It can be "enum like" string.
Current possible values are: `ga`, `beta` or missing.

Reply:
Sure

Reply:
Changed, PTAL again